### ISSUE TITLE
Hazelcast does not startup with tomcat9 sessionmanager

### ISSUE THREAD

**Issue by bluegodmsk** [hazelcast/hazelcast-tomcat-sessionmanager#85]
Currently we are using the Tomcat8.5 with the tomcat85-sessionamanger-1.1.3.jar and hazelcast-all-3.12.1. Everything runs smoothly and the Hazelcast cluster is formed on Tomcat startup. Session replication works.

Now we need to upgrade the Tomcat-9.0.33. So, I have tried bringing up the hazelcast cache with the following combinations of tomcat9-sessionmanager and hazelcast-all jars. However, the hazelcast cache does not startup on tomcat startup. No errors are logged in the tomcat logs. 

Tried the following combinations:

tomcat9-sessionmanager-2.0 with hazelcast-all-4.0
tomcat9-sessionmanager-1.1.5 with hazelcast-all-3.12.1

However, strangely, if I start the tomcat 9 with the following combination it starts up fine:
tomcat85-sessionmanager-1.1.3 with hazelcast-all-3.12.1

I do not want to use the 8.5 combination on Tomcat 9.0 if it is not recommended. 
Is there a particular version of tomcat 9.x that the tomcat9-sessionmanager jar works with?

**Comment by alparslanavci** [comment]
Hi @bluegodmsk, I tried both combinations below for P2P deployment and Hazelcast member is started in both setups by using Tomcat-9.0.33.

```
tomcat9-sessionmanager-2.0 with hazelcast-all-4.0
tomcat9-sessionmanager-1.1.5 with hazelcast-all-3.12.1
```

Can you please confirm that there are no other configuration changes other than the lib changes mentioned?

**Comment by bluegodmsk** [comment]
Thanks for checking let me try again see if I can find more.

**Comment by alparslanavci** [comment]
@bluegodmsk, do you have any updates about this issue?

**Comment by manish-in-java** [comment]
I am able to start Tomcat 9 with Hazelcast 4.1 and Tomcat9 SessionManager 2.1. I have a sample application at https://drive.google.com/file/d/1dSjgwGyUqJe6cd3-QnytKJf6lvBiFv2L/view?usp=sharing that can be tried.

**Comment by nfrankel** [comment]
@alparslanavci I think this issue can be closed for lack of activity.
